src/lib.rs

```diff
@@ -510,11 +645,14 @@ impl GooseAttack {
             test_stop_task: None,
             task_sets: Vec::new(),
             weighted_users: Vec::new(),
-            host: None,
+            defaults: GooseDefaults::default(),
             configuration: GooseConfiguration::parse_args_default_or_exit(),
             number_of_cpus: num_cpus::get(),
             run_time: 0,
             users: 0,
+            hatch_rate: 0,
+            throttle_requests: 0,
+            attack_mode: GooseMode::Undefined,
```

Comment:
@LionsAd Here are the three new things that converged into `GooseAttack`, instead of over-loading configuration.

Reply:
I re-removed `hatch_rate` and `throttle_requests` and those are fully managed in `configuration`. Now only `attack_mode` is newly added to `GooseAttack`.